Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "winit"
-version = "0.9.0"
+version = "0.10.0"
```

Comment:
Please do not randomly bump the version in winit.

Reply:
Oh oops. Forgot I'd committed that in the PR. I'll remove it. I did it when I was working on getting jwilm/alacritty to use my branch.